contracts/test/unit/VoltV2GovCompatibility.t.sol

```diff
@@ -0,0 +1,196 @@
+    }
+
+    function testUserWithNoVoltCanPropose() public {
+        (
+            address[] memory targets,
+            uint256[] memory values,
+            bytes[] memory calldatas,
+            string memory description,
+
+        ) = _createDummyProposal();
+
+        uint256 noVoltProposalId = mockDAO.propose(
+            targets,
+            values,
+            calldatas,
+            description
+        );
+
+        assertEq(uint8(mockDAO.state(noVoltProposalId)), 0); // Pending
+        vm.roll(block.number + 1);
+
+        assertEq(uint8(mockDAO.state(noVoltProposalId)), 1); // Active
+    }
+
+    function testProposalExecutes() public {
+        (
+            address[] memory targets,
+            uint256[] memory values,
+            bytes[] memory calldatas,
+            string memory description,
+            bytes32 descriptionHash
+        ) = _createDummyProposal();
+
+        vm.prank(addresses.minterAddress);
+        voltV2.mint(userWithVolt, quorum);
+
+        vm.prank(userWithVolt);
+        voltV2.delegate(userWithVolt);
+
+        uint256 proposalId = mockDAO.propose(
+            targets,
+            values,
+            calldatas,
+            description
+        );
+
+        // Advance past the 1 voting block
+        vm.roll(block.number + 1);
+
+        // Cast a vote for the proposal, in excess of quorum
+        vm.prank(userWithVolt);
+        mockDAO.castVote(proposalId, 1);
+
+        vm.roll(block.number + 2);
+
+        vm.startPrank(proposerCancellerExecutor);
+        mockDAO.queue(targets, values, calldatas, descriptionHash);
+        vm.stopPrank();
+
+        vm.warp(block.number + timelock.getMinDelay());
+
+        vm.startPrank(proposerCancellerExecutor);
+
+        // Execute
+        mockDAO.execute(targets, values, calldatas, descriptionHash);
+
+        assertEq(mockToken.balanceOf(userWithVolt), 1_000_000e18);
+    }
+
+    function testProposalRejectsIfNotQuorum() public {
+        address userWithInsufficientVolt = address(0xFFF);
+        (
+            address[] memory targets,
+            uint256[] memory values,
+            bytes[] memory calldatas,
+            string memory description,
+            bytes32 descriptionHash
+        ) = _createDummyProposal();
+
+        vm.prank(userWithInsufficientVolt);
+        uint256 proposalId = mockDAO.propose(
+            targets,
+            values,
+            calldatas,
+            description
+        );
+
+        vm.prank(addresses.minterAddress);
+        voltV2.mint(userWithInsufficientVolt, quorum - 1);
+
+        vm.prank(userWithInsufficientVolt);
+        voltV2.delegate(userWithInsufficientVolt);
+
+        // Advance past the 1 voting block
+        vm.roll(block.number + 1);
+
+        // Cast a vote for the proposal, in excess of quorum
+        vm.prank(userWithInsufficientVolt);
+        mockDAO.castVote(proposalId, 1);
+
+        vm.roll(block.number + 2);
```

Comment:
assert correct number of votes, assert that quorum was not reached, assert mocktoken balances unchanged